examples/multisig_transfer.ts

```diff
@@ -0,0 +1,150 @@
+// FIXME: remove this check once the Zones .bind(env) fix is merged
+const hostname = Deno.env.get("TEST_CTX_HOSTNAME");
+const portRaw = Deno.env.get("TEST_CTX_PORT");
+if (!hostname || !portRaw) {
+  throw new Error("Must be running inside a test ctx");
+}
+
+import { createKeyMulti } from "../deps/polkadot/util-crypto.ts";
+import * as C from "../mod.ts";
+import * as T from "../test_util/mod.ts";
+import * as U from "../util/mod.ts";
+
+type KeyringPair = typeof T.alice;
```

Comment:
Please delete this line and instead specify the `Uint8Array` and `C.M.SignExtrinsic` types where applicable.